tests/versioned/common/transactions.js

```diff
@@ -14,20 +14,23 @@ const params = setup.params
 
 module.exports = (t, requireMySQL) => {
   t.test('MySQL transactions', {timeout: 30000}, (t) => {
```

Comment:
refactored this to make it easier to await setup and not deal with wrapping the rest in an async function to properly handling the nested suites